eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -198,6 +206,23 @@ jobs:
         displayName: 'product build'
 
 
+    - ${{ if eq(parameters.runtimeFlavor, 'mono') }}:
+      # We need to explictly download CoreCLR for Mono
```

Comment:
DO we need this in order to build the test wrappers? I don't remember why we needed it in `run-test-job` and `build-test-job`

Reply:
Yes; its required in build-test because its needed to build the test stuff. It's required in run-test because we need to patch it with mono-bits.

Reply:
Oh yeah, I forgot about that. Thanks.